provisioning/service/samples/readme.md

```diff
@@ -0,0 +1,72 @@
+# Samples for the Azure IoT Device Provisioning Service SDK for Node.js
+
+This file can be found in http://github.com/azure/azure-iot-sdk-node/provisioning/service/samples
+
+This folder contains simple samples showing how to use the various features of the Microsoft Azure IoT Device Provisioning Service from an application running JavaScript code.
+
+## List of samples
+
+* Create a simple IndividualEnrollment object.
+   *  **simple_IndividualEnrollment_create.js**
```

Comment:
>**simple_IndividualEnrollment_create.js** [](start = 6, length = 41)

nit: these could easily be turned into links which would be slightly nicer.

Reply:
Done.

---
In reply to: [150136233](https://github.com/Azure/azure-iot-sdk-node/pull/142#discussion_r150136233) [](ancestors = 150136233)